### ISSUE TITLE
Missing Keyboard Shortcuts documentation

### ISSUE THREAD

**Issue by Framsfex** [linuxmint/pix#198]
### Distribution

Mint 21.2

### Package version

3.0.2

### Frequency

Always

### Bug description

I am using Pix 3.0.2 on Mint 21.2
In the "Keyboard Shortcuts" help some keys are missing, for example [Shift] [C] for crop, [R] for rotate or [I] for properties.
I found them only incidentally.
And is there a shortcut for "Fit to window"?

### Steps to reproduce

[F10] -> "Keyboard Shortcuts"

### Expected behavior

Show documentation of all actions bound to keys

### Additional information

I am coming from "xv", an UNIX image viewer with a great usability :-)

**Comment by Framsfex** [comment]
Meanwhile I have written my own shortcuts documentation: an xmessage helper script
https://fex.belwue.de/sw/share/fstools-0.0/bin/pixkeys

(How) can I bind it to a key inside pix?

**Comment by fr4g3** [comment]
The shortcuts you have mentioned are certainly there in the shortcuts documentations. And yes Shift+x is the shortcut for "fit to window"
![image](https://github.com/linuxmint/pix/assets/95216756/155e19e8-dae3-49c6-9e0e-652e08f26339)

**Comment by Framsfex** [comment]
Thanks for the screenshot, it shows at the bottom 4 page numbers which I did not see and explains my misunderstanding!
XFCE hides it for me, see my screenshot. I did not know that I have to move up the keyboard documentation window!
![X](https://github.com/linuxmint/pix/assets/33547527/febe8d15-7207-49b3-937c-b5f11bbc23d1)

Another feature request: it would be great to change the document pages with the cursor keys right/left like in the image viewer main window.
